api/src/main/java/org/openmrs/util/LocaleUtility.java

```diff
@@ -59,13 +59,8 @@ public static Locale getDefaultLocale() {
 					String locale = Context.getAdministrationService().getGlobalProperty(
 					    OpenmrsConstants.GLOBAL_PROPERTY_DEFAULT_LOCALE);
 					
-					if (StringUtils.hasLength(locale)) {
-						try {
-							defaultLocaleCache = fromSpecification(locale);
-						}
-						catch (Exception t) {
-							log.warn("Unable to parse default locale global property value: " + locale, t);
-						}
+					if (StringUtils.hasLength(locale) && isValid(fromSpecification(locale))) {
```

Comment:
This shouldn't require two calls to `fromSpecification()`. However, this PR is also not dealing with the issue in the ticket, which is about the _user's_ default locale property, not the property from the global property. The change here is unnecessary because we validate the global property when setting it.